web-console/knikubevirt/Create-vm/wizard-import/wizard-import.md

```diff
@@ -0,0 +1,73 @@
+# Create A Virtual Machine
+
+## Import VM
+
+The ‘Import virtual machine’ action will be available in the ‘Create virtual machine’ dropdown.
+
+![Empty state](img/empty-state.png)
+
+![wizard default screen](img/default.png)
```

Comment:
Similar to the Create VM workflow, should the action be "Review and import virtual machine"? The "review" step could still have the action to "Import virtual machine"

Reply:
Good call @rileyhuston although to align with the create flow it should just be "Review and import"